webhooks/webhooks.constant.ts

```diff
@@ -0,0 +1,91 @@
+export class WebhooksConstant {
+    static readonly webhooksUrl = '/f/page?ixPlugin=21&pg=pgPluginConfig';
+
+    static readonly attributes = Object.freeze({
+        text: Object.freeze({
+            header: 'Webhooks',
+            addNewHook: 'Add New Hook',
+            editHook: 'Editing Hook',
+            deleteHook: 'Delete Hook',
+            ok: 'o',
+            viewLog: 'View Log',
+            import: 'Import',
+            export: 'Export',
+            xmlImport: 'XML Import',
+            importOk: 'OK',
+            importSuccessfulMessage: 'Import Successful: Webhooks.xml',
+            returnToConfigurationPage: 'Return to Configuration Page'
+        }),
+        class: Object.freeze({
+            headerClass: 'headline',
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/davidgunadi/hello-world/pullRequest?prid=8400130)
```suggestion
            headerClass: "headline",
```